### PR TITLE
Add missing content to What's New 7.14

### PR DESCRIPTION
**Pull request by ollyhowell**

### CODE REVIEW

File: docs/en/observability/whats-new.asciidoc [hunk 1]
```diff
@@ -128,4 +128,49 @@ Elastic Observability. For a holistic view of your deployments, you can also qui
...
+
+As it is virtually impossible to manually analyze what has become an overwhelming amount of data and dependencies in modern applications,
+automation and analytics have become essential components of any troubleshooting toolkit. Elastic’s APM correlations feature (beta)
+accelerates root cause analysis by automatically surfacing attributes of your data set (such as infrastructure components, versions, and
+locations) that are correlated with high-latency or erroneous transactions and have the most significant impact on your service performance.
+You can visualize the latency distribution of any attribute compared to overall latency and use these attributes to filter and isolate the root
+causes of your performance problems.
+
+[role="screenshot"]
+image::images/apm-correlations.png[APM correlations]
```
Comment (dedemorton) [inline docs/en/observability/whats-new.asciidoc:149]: Looks like the doc build is failing because some images are missing.

**Review by dedemorton — COMMENTED** [review]
Just a few minor comment. Otherwise looks good.

File: docs/en/observability/whats-new.asciidoc (hunk 1 above)
Comment (ollyhowell) [reply docs/en/observability/whats-new.asciidoc:149]: Ah, good catch. I redid my changes from a clean branch and forgot to stage the images too.

File: docs/en/observability/whats-new.asciidoc [hunk 2]
```diff
@@ -128,4 +128,49 @@ Elastic Observability. For a holistic view of your deployments, you can also qui
...
+
+[discrete]
+== APM Mobile - iOS support (technical preview)
+
+Native mobile applications are increasingly important to modern businesses. We are happy to announce that this release introduces a technical
```
[the remark this answers is not in the document]
Comment (ollyhowell) [reply docs/en/observability/whats-new.asciidoc:154]: Agree

File: docs/en/observability/whats-new.asciidoc [hunk 3]
```diff
@@ -128,4 +128,49 @@ Elastic Observability. For a holistic view of your deployments, you can also qui
...
+
+Histogram support for OpenTelemetry agents enables you to profile the distribution of measurements for user analytics and troubleshooting.
+You can now identify subpopulation patterns in your traffic with granular analysis of OpenTelemetry time series data, such as transaction latency, 
+payload size, and business KPIs.
+
+[discrete]
+== APM transaction correlations (beta)
+
+As it is virtually impossible to manually analyze what has become an overwhelming amount of data and dependencies in modern applications,
+automation and analytics have become essential components of any troubleshooting toolkit. Elastic’s APM correlations feature (beta)
```
[the remark this answers is not in the document]
Comment (ollyhowell) [reply docs/en/observability/whats-new.asciidoc:142]: Yeah, also agree - I stumbled over this a bit too and should have got the red pen out.

**Comment by ollyhowell** [comment]
Did you want to take a quick look or nominate someone @tbragin?
There is a preview here somewhere... https://observability-docs_1293.docs-preview.app.elstc.co/diff

**Comment by tbragin** [comment]
@ollyhowell LGTM!

**Pull request merged**